assets/javascripts/application.js

```diff
@@ -5,20 +5,8 @@
 $(document).ready(function () {
   branchCoverageActivated = $("body").data("branch-coverage") == true;
 
-  // Configuration for fancy sortable tables for source file groups
-  // For what these columns are, cecl out file_list.erb - null basically
-  // does normal/number style sorting which fine for hit/missed lines only
-  // percentages like "covered percent" need special sorting arguments.
-  lineColumns = [null, { "sType": "percent" }, null, null, null, null, null];
-  branchColumns = [{ "sType": "percent" }, null, null, null];
-
-  dataColumns = branchCoverageActivated ? lineColumns.concat(branchColumns) : lineColumns;
-
   $('.file_list').dataTable({
-    "aaSorting": [[1, "asc"]],
-    "bPaginate": false,
-    "bJQueryUI": true,
-    "aoColumns": dataColumns
+    "aaSorting": [[1, "asc"]]
```

Comment:
Are you sure that this is still a valid option in the API? I can only find it in the legacy docs https://legacy.datatables.net/ref#aaSorting

Reply:
:wave: 

I'm not. However it still sorted the column I wanted to ascending so I was like :man_shrugging: "maybe a hidden compatibility layer"

I can try removing it, maybe it's another "default magic" thing from datatables